README.md

```diff
@@ -9,20 +9,24 @@ speaks, see:
 Running locally
 ---------------
 
-For dev testing you probably want to run from a virtualenv::
+For dev testing, create and activate a virtual environment:
 
-    virtualenv shavar
+    virtualenv -p python3.7 shavar
     source shavar/bin/activate
 
 Install the necessary dependencies:
 
-    pip install -r requirements.txt
+    virtualenv -p python3.7 shavar
```

Comment:
is this supposed to change, or is this a bad copy-paste?

Reply:
That's a bad copy-paste from my personal note to the README 🤦‍♂. Updated [here](https://github.com/mozilla-services/shavar/pull/131/commits/d24b91cbdcce32f287668a687f88f6e3017193bc).